src/eredis_client.erl

```diff
@@ -370,7 +370,7 @@ transport_params(State) when State#state.transport =:= tcp ->
 transport_params(State) when State#state.transport =:= ssl ->
     Host = State#state.host,
     Port = State#state.port,
-    ConnectOpts = eredis_security:secure_ssl_opts(Host, ?SSL_SOCKET_OPTS),
```

Comment:
what other purpose does `eredis_security`serve?

Reply:
None whatsoever, that's why it was removed.

Reply:
right, missed it